README.rst

```diff
@@ -87,6 +87,18 @@ conftest.py file:
               extra.append(pytest_html.extras.html('<div>Additional HTML</div>'))
           report.extra = extra
 
+Creating a single file to the project
```

Comment:
Great idea to add documentation for this. I would move it to before the 'Enhancing reports' section though as it is less advanced than that. I would also change the header to 'Creating a self-contained report'.